notebook/static/notebook/js/notebook.js

```diff
@@ -113,7 +113,7 @@ define(function (require) {
                         }
                     }, function (err) {
                         console.log("No CodeMirror mode: " + lang);
-                        callback(err, code);
+                        callback(null, code);
```

Comment:
I'm unsure if it's the desired behavior to suppress the error here.
I think [marked](https://github.com/chjj/marked) doesn't produce a html output when `err != null` in this callback.